src/lib/openjp2/CMakeLists.txt

```diff
@@ -87,7 +87,9 @@ if(WIN32)
     add_definitions(-DOPJ_STATIC)
   endif()
 endif()
-add_library(${OPENJPEG_LIBRARY_NAME} ${OPENJPEG_SRCS})
+add_library(${OPENJPEG_LIBRARY_NAME} SHARED ${OPENJPEG_SRCS})
+add_library(openjp2_static STATIC ${OPENJPEG_SRCS})
```

Comment:
Any reason not use ${OPENJPEG_LIBRARY_NAME} as well ? Generally both static and dynamic library have the same basename.